### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .test_config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "test_package": true
```
Comment (natebosch) [inline .test_config]: what is this file?

Comment (kevmoo) [reply .test_config]: picked up by the package bots

**Comment by kevmoo** [comment]
> Please add more detail to the commit message

See description in the two commits

**Comment by natebosch** [comment]
> See description in the two commits

I see "misc: fix tests" which doesn't tell me much 😛

**Comment by kevmoo** [comment]
@natebosch – PTAL

**Pull request merged**